fdbclient/admin_client_test.go

```diff
@@ -921,8 +921,39 @@ protocol fdb00b071010000`,
 				Expect(err).NotTo(HaveOccurred())
 			})
 
-			It("should not issue an exclude command", func() {
-				Expect(mockRunner.receivedBinary).To(BeEmpty())
+			It("should issue an exclude command to verify the exclusion", func() {
+				Expect(mockRunner.receivedBinary).To(HaveSuffix(fdbcliStr))
+				Expect(mockRunner.receivedArgs).To(ContainElements("exclude 192.168.0.1:4500 192.168.0.2:4500"))
+			})
+		})
+
+		When("all provided processes are fully excluded in the status but the exclude command returns an error", func() {
```

Comment:
Dumb question, but does this check the situation when there is no process with this IP address serving a role, but we still get a timeout from the exclude command? It's what I think this does, but want to confirm.

Reply:
Correct. This "simulates" a case where a process is in the machine-readable status and has no roles but the exclude command times out because the process is not fully excluded.